trainings/admin-webinar/lesson-plan.md

```diff
@@ -276,8 +287,66 @@ User activity can be tracked in-app and via the API. This is the basis for user
+
+🔎 The trainer should demonstrate:
+
+* How to access the Search Analytics graph
+* How to change the timeframe for the graph (up to 3 months of data)
+* How to customize the time saved data below the graph
+
+❗️ Ideally, during the PoC phase you will have discussed important internal metrics with the user (do they care about time saved, ability to complete complex searches, etc). If you have, use that to inform how you discuss the analytics.
+
+### Viewing and using Code Intel Analytics
+
+The Code Intel section of Analytics operates similarly. Code Intel or Code Navigation is our way of finding references to symbols as well as traveling to the definition, and it works cross-repo. You'll see how many `find references` and `go to definition` events have been triggered by your users, and how many of those were cross-repo. You'll see a month of data by default but can change that on the top right.
+
+🔎 The trainer should demonstrate:
+
+* How to access the Code Intel graph
+* How to customize the time saved data below the graph
+
+❗️ Highlight that cross-repo events generally aren't possible without Sourcegraph, particularly if the user is working in a microservices architecture.
+
+❗️ Highlight the benefits and `compiler-accurate` language for LSIF/SCIP, if the customer is using them.
+
+### Viewing and using Batch Change Analytics
+
+❗️Skip this section if the customer isn't using Batch Changes. 
+
+The Batch Changes section of the Analytics tracks the impact of changesets open and merged using the Batch Changes functionality, which allows you to open and manage changesets en masse. You can easily see the number of changesets (PRs, MRs) opened with Sourcegraph, and the number ultimately merged. You can track the amount of time that the merged changesets have saved your team.
+
+🔎 The trainer should demonstrate:
+
+* How to access the Batch Changes graph
+* How to customize the time saved data
+
+### Viewing and using Notebooks Analytics
+
+Notebooks in Sourcegraph allow team members to create living documentation collections of searches, references to files, and explanations for why/why not to do something in the code. The Notebooks section of Analytics shows the number of notebooks created, the number of views of them, and the number of times an embedded block of live-updated content is "run" (updated). Similar to the other panels, you can track the amount of time this saves your team when onboarding a new teammate or sharing context.
+
+🔎 The trainer should demonstrate:
+
+* How to access the Notebooks graph
+* How to customize the time saved data
+
+### Conclusion
+
+The Analytics functionality will allow your team to track time saved and show the value Sourcegraph brings to your team.
+
+### Resources
+
+* TBD once they are released
```

Comment:
We're iterating on [this page](https://docs.sourcegraph.com/admin/admin_analytics). Is this type of resources you're looking for?

Reply:
Yes! Wasn't coming up in my initial Docs search. Will add.